paasta_tools/utils.py

```diff
@@ -2556,6 +2558,16 @@ def get_legacy_autoscaling_signalflow(self) -> str:
             "legacy_autoscaling_signalflow", default_legacy_autoscaling_signalflow
         )
 
+    def get_uwsgi_exporter_sidecar_image_url(self) -> str:
+        """Get the docker image URL for the uwsgi_exporter sidecar container"""
+        return self.config_dict.get(
+            "uwsgi_exporter_sidecar_image_url",
+            "docker-paasta.yelpcorp.com:443/uwsgi_exporter-k8s-sidecar:v1.0.0",
+        )
+
+    def default_should_run_uwsgi_exporter_sidecar(self) -> bool:
+        return self.config_dict.get("default_should_run_uwsgi_exporter_sidecar", False,)
```

Comment:
nit: extra comma at the end?  Doesn't look like it changes things, though.

Reply:
huh, surprised that Black let that stay.